Source/Core/Core/Lua/LuaFunctions/LuaRegisters.cpp

```diff
@@ -0,0 +1,467 @@
+#include "LuaRegisters.h"
```

Comment:
We don't use direct or folder relative includes for source files.

These should be written as if from the folder base directory like:

```cpp
#include "Core/Lua/LuaFunctions/LuaRegisters.h"
```

etc.

Ditto for all occurrences